### DISCUSSION TITLE
Carrousel funcionando em localhost, mas dando problema no github page

### DISCUSSION THREAD

**Discussion by 19733** [frontendbr/forum#2223] [Perguntas e Respostas] [answered]
Meu carrousel de imagens funciona normalmente em localhost, porem quando abro meu site no github page as imagens não se mexem,  não sei porque.

Segue o link do meu site: https://19733.github.io/juandeveloper/

**Comment by carloschneider** [comment, marked as answer]
@19733 a sua página está rodando com https e você está importando o jquery sem https.

![image](https://user-images.githubusercontent.com/577143/147495359-5a20aedb-9ec9-4281-9267-ced9ca4ea1ea.png)

Acredito que seja isso.

**Reply by carloschneider** [reply]
Para visualizar esse erro eu abri o `Console` do `Chrome Developer Tools` usando: <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>j</kbd>

**Reply by 19733** [reply]
não está funcionando msm assim, fiz errado?

<script src="https://19733.github.io/juandeveloper/.jquery.com/jquery-latest.js"></script>

**Reply by 19733** [reply]
Opa agr deu certo colega, entendi o que voce quis dizer, muito obrigado
